src/components/common/names-capture.vue

```diff
@@ -8,7 +8,7 @@
         </v-row>
 
         <v-row class="mt-3">
-          <v-col cols="5" class="py-0">
+        <v-col cols="5" class="py-0">
```

Comment:
i believe you still want an indent here as the `v-col` is a child element of the `v-row`.